models/devicesim-1.0.x/VERSION

```diff
@@ -1 +1 @@
-0.5.22
+0.5.23-dev
```

Comment:
I think my patch has reset these VERSIONs already

Reply:
@SeanCondon Yes, I added those just so the build would pass.